src/pages/library/common.js

```diff
@@ -192,8 +195,32 @@ export const selectionActionComponent = (selectedData, setSelectedData) => {
           }, 'Cancel'),
           h(ButtonPrimary, {
             style: { textTransform: 'none', fontSize: 14 },
-            onClick: () => {}
+            onClick: () => {
+              setShowExportModal(true)
+            }
           }, ['Save to a workspace'])
+        ]),
+        showExportModal &&
+        h(Modal, {
```

Comment:
We don't newline for these cases:

```suggestion
        showExportModal && h(Modal, {
```